robocluster/__init__.py

```diff
@@ -1,2 +1,50 @@
-def request(uri):
-    raise NotImplementedError()
+"""Message passing for robocluster."""
+
+from collections import defaultdict
+
+from .net import Link, MulticastReceiver
+
+
+class Device:
```

Comment:
Totaly pedantic thing, but to me "Device" is a microcontroller, GPS unit, etc. This class could be used in a program that publishes information about the navigation system, or handling the dynamics of the arm movement. Would "Service" be a more suitable term?